### ISSUE TITLE
Discussion for MASVS lvl 1 & 2

### ISSUE THREAD

**Issue by commjoen** [OWASP/owasp-masvs#277]
After gathering feedback from various places, I have heared that people find it hard to just implement the list of controls. So what if we say that all LVL 2 controls should be based on a threatmodel? (and selected when you need it?)
Similarly we need to make sure to have a proper rationale for the lvl1 requirements. This is however, something the new mobile threatmodel project will take care off (shoutout to @Tommythetipper ;-)).

**Comment by Tommythetipper** [comment]
Absolutely! That’s the idea - if anyone wants to help out, please ping me!

**Comment by sushi2k** [comment]
We are explaining the usage of Level 1/2 in the MSTG in this section for penetration tests:

https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04b-Mobile-App-Security-Testing.md#preparation

As part of the SDLC it's explained here. Even though that we are saying Level 1 is mandatory for all mobile apps, this should always be decided on an individual basis according to the threat model and business requirements of the app and internal policies. If this is not clearly articulated in the MSTG, we should maybe rephrase it. The intention was never to implement blindly all level 1 requirements, but only the ones that are applicable (which might be a mix of Level 1 and 2) for the app. 

https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04b-Mobile-App-Security-Testing.md#sdlc-overview

**Comment by TheDauntless** [comment]
How we used it, the goal is to assess a risk. It is then up to "business" to accept the risk or mitigate it.

In that sense, maybe the threat model project should indicate what the risk is of not implementing a control for each MASVS entree ?

**Comment by commjoen** [comment]
The idea of the MRP (Mobile Risk Project ^^ @Tommythetipper ) should indeed show which risks are countered (e.g. impact reduction/ chance reduction) for each control. 
@sushi2k : its great that we are explaining it in the mstg, but we should do this as well wat the MASVS. People often just take one document and run with it :D... AFter all: the masvs is much smaller than the mstg ;-)

**Comment by sushi2k** [comment]
You are right and thanks for PR 277! Already approved it.

**Comment by commjoen** [comment]
Fixed in #277.
